### DISCUSSION TITLE
Has anyone created a scrollable table using Chakra?

### DISCUSSION THREAD

**Discussion by Sandilly** [chakra-ui/chakra-ui#7197] [Q&A]
I am trying to create a scrollable table. I want the table headers to be fixed and only the rows to be scrollable. But in order for me to do this. I had to set a height and an overflow on `<Tbody>`, and have add `display: block` on it which throws off all the alignment. Has anyone been able to achieve this yet?

**Comment by hellenfc** [comment]
```
<TableContainer>
    <div style={{ maxHeight: '300px', overflowY: 'auto' }}>
          <Table >
              <Tbody>
              </Tbody>
          </Table>
     </div>
</TableContainer>
```

**Comment by davidgreenspan15** [comment]
The work around I found was to make the header row sticky
1. TableContainer should have `max height`, `overflowY:"auto"`, and `position:"relative"`
2. Thead should have `position: "sticky"` and `top: 0`

Note: Thead will need a background color or filter since it will be over the other rows
